gst/tensor_ros_src/tensor_ros_src.cc

```diff
@@ -388,6 +403,101 @@ gst_tensor_ros_src_dispose (GObject * object)
   G_OBJECT_CLASS (parent_class)->dispose (object);
 }
 
+/**
+ * @brief Start handler
+ */
+static gboolean
+gst_tensor_ros_src_start (GstBaseSrc * src)
+{
+  GstTensorRosSrc *rossrc = GST_TENSOR_ROS_SRC (src);
+
+  silent_debug (rossrc, "start RosSubscriber");
+  rossrc->ros_sub = new TensorRosSub ("TensorRosSub", rossrc->topic_name,
+    rossrc, rossrc->freq_rate);
+  rossrc->ros_sub->Start (&rossrc->thread);
+
+  while (!rossrc->configured)
```

Comment:
should there be a max timeout limit, so that the code does not get stuck here if there is no callback for RosListener?

Reply:
OK. I will adds the property about `timeout` and `input' dimension for this issue as another PR.
Thanks.